.github/workflows/docker-deploy.yml

```diff
@@ -13,8 +13,8 @@ on:
       - labeled
 
 env:
-  REGISTRY_IMAGE: ${{ secrets.DOCKER_USERNAME }}/AQUÍ_VA_EL_NOMBRE_DE_TU_IMAGEN_SIN_MAYUSCULAS
-  GHCR_IMAGE: fis2425/AQUÍ_VA_EL_NOMBRE_DE_TU_IMAGEN_SIN_MAYUSCULAS
+  REGISTRY_IMAGE: ${{ secrets.DOCKER_USERNAME }}/authorization-svc
```

Comment:
To follow the same naming strategy, I recommend using only the name of the service (Example: `authorization`). You can check existing images [here](https://hub.docker.com/repositories/cloudmedix)

Reply:
that makes sense. Will do

Reply:
Do you want me to amend the previous commit or create a new one?